src/broadcast/setup.ts

```diff
@@ -56,6 +106,10 @@ export default function setupBroadcastIpc({
     );
 
     await spectateWorker.refreshBroadcastList();
+
+    // Reset idle timeout on refresh
+    await resetSpectateIdleTimeout();
```

Comment:
only reset spectate idle timeout if idle timeout has been started. It may have been stopped below because we started watching a broadcast, in which case we don't want to reset (implicitly start) it here